rules/branch-naming/rule.md

```diff
@@ -0,0 +1,78 @@
+---
+type: rule
+tips: ""
+title: Do your branches have a naming convention?
+seoDescription: Using effective branch naming conventions with PBI numbers, descriptive names, and structured formats to enhance collaboration and traceability.
+uri: branch-naming
+authors:
+  - title: Christian Morford-Waite
+    url: https://ssw.com.au/people/christian-morford-waite
+related:
+  - do-you-know-when-to-branch-in-git
+  - clear-meaningful-names
+created: 2025-01-02T15:40:00.000Z
+guid: a97fa64d-7684-4220-8042-0b3be889aa9d
+---
+
+Have you ever looked at the list of branches in your repository and not know what they were for?
+
+Consistent naming conventions in source control systems help improve collaboration, clarity, and organization across teams. Using meaningful branch names with PBI numbers enhances traceability and reduces [technical debt](/technical-debt).
```

Comment:
> and reduces technical debt.

Not sure how that works... It's just a branch name - how does that add to technical debt? Do you mean *merge debt*?